src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/matmul_decompress_convert.cpp

```diff
@@ -207,6 +207,13 @@ class MatMulDecompressConvertTest : public testing::WithParamInterface<MatMulDec
         }
 
         std::string cpuNodeType = "FullyConnected";
+        // replace kleidiai with acl type if input shapes are not 2D
+#if defined(OPENVINO_ARCH_ARM64)
+        if (selectedType == "kleidiai" &&
+            (inShapeA.rank().get_length() != 2 || inShapeB.rank().get_length() != 2)) {
+            selectedType = "acl";
+        }
+#endif
```

Comment:
Since we are lowering priority of kleidiai impl, this WA is not required anymore.

Reply:
Removed